cdc_replication_test.py

```diff
@@ -71,12 +75,25 @@ def collect_data_for_analysis(self,
 
     # pylint: disable=too-many-statements,too-many-branches,too-many-locals
     def test_replication(self) -> None:
+        gemini_test = is_gemini_test(os.environ)
+        self.log.info('Using {} to generate workload.'.format('gemini' if gemini_test else 'cassandra-stress'))
+
         self.log.info('Waiting for the latest CDC generation to start...')
         # 2 * ring_delay (ring_delay = 30s) + leeway
         time.sleep(70)
 
         self.log.info('Starting stressor.')
-        stress_thread = self.run_stress_cassandra_thread(stress_cmd=self.params.get('stress_cmd'))
+        if gemini_test:
+            stress_thread = GeminiStressThread(
+                    test_cluster=self.db_cluster,
+                    oracle_cluster=None,
+                    loaders=self.loaders,
+                    gemini_cmd=self.params.get('gemini_cmd'),
+                    timeout=self.get_duration(None),
+                    outputdir=self.loaders.logdir,
+                    params=self.params).run()
+        else:
+            stress_thread = self.run_stress_cassandra_thread(stress_cmd=self.params.get('stress_cmd'))
```

Comment:
I think we don't need to the additional config params.
It will enough check param "gemini_cmd" or "cassandra_stress"